### DISCUSSION TITLE
porting on ipad

### DISCUSSION THREAD

**Discussion by u-deep** [wxMaxima-developers/wxmaxima#1891] [Ideas]
in future there is the possibility to have an iPad version?

**Comment by daute** [comment]
Probably not very likely in the near future:
(a) For wxMaxima (which is based on wxWidgets), the wxWidgets port is far from a full supported port (https://docs.wxwidgets.org/latest/plat_ios_install.html).
One would also need a working Maxima (the computeralgebrasystem behind wxMaxima), which is written in Lisp, don't know if that works on IOS. (There is a port of Maxima (not wxMaxima, but a great App) to Android).

(b) The terms of Apple for it's "App-Store" seems to collide with the  GPL (a very usual free software license, which is the License for Maxima, wxMaxima, ...):
https://www.fsf.org/news/2010-05-app-store-compliance
(that was from 2010, don't know, if Apple allows free software now, I am not a lawyer)

But if you are an developer and can contribute to wxMaxima, we welcome your contributions!
